test/geometry/scalingtest.cpp

```diff
@@ -12,7 +12,7 @@ TEST_CASE("Geometry: scaling a Point", "[geometry][operations]") {
     PointFixture pointFixture;
 
     Point scaledPoint;
-    scale(pointFixture.point, scalingPoint, scaledPoint);
+    scale(pointFixture.point, scalingPoint.x(), scalingPoint.y(), scaledPoint);
```

Comment:
Actually, I think this could be fixed by adding another template to the Point type in the geometric operations, I'll try that.